### PR TITLE
Deprecating the SCCP-documentation

### PR DESCRIPTION
**Pull request by jacobcrawford**
Mark the project as deprecated

### CODE REVIEW

**Review by MadsHT — COMMENTED** [review]
LGTM 🎉

**Review by MadsHT — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 ![Energinet logo](images/Energinet-logo.png)
 
+# Deprecation warning
+This project is no longer under maintenance as the project is under evaluation. This mean that the open source parts of the project is paused and might be resumed in the future, without gurantee.
```
Comment (klausenbusk) [inline README.md:4]: Typo
```suggestion
This project is no longer under maintenance as the project is under evaluation. This mean that the open source parts of the project is paused and might be resumed in the future, without guarantee. 
```

**Review by klausenbusk — DISMISSED** [review]
LGTM 🎉

**Review by klausenbusk — APPROVED** [review]
LGTM 🎉

**Review by nashton — APPROVED** [review]

**Pull request merged**